examples/archive.html

```diff
@@ -91,39 +90,60 @@ <h3 id="offcanvasRightLabel">Images</h3>
 
     setupMap();
     setupCollection();
-    
+
     form.addEventListener('submit', (event) => {
       event.preventDefault();
       const url = input.value.replace('details', 'metadata');
-      axios.get(url)
+      let fetchedFrom;
+      axios
+        .get(url)
         .then((response) => {
           if (response.data.files && response.data.files.length != 0) {
             let imageCount = 0;
-            response.data.files.forEach(file => {
-              if (file.format === 'PNG' || file.format === 'JPEG'){
+            response.data.files.forEach((file) => {
+              if (file.format === 'PNG' || file.format === 'JPEG') {
                 let imageRow = document.createElement('div');
                 let image = new Image(150, 150);
                 let placeButton = document.createElement('a');
-
-                placeButton.classList.add('btn', 'btn-sm', 'btn-outline-secondary', 'place-button');
+                fetchedFrom = document.createElement('p');
+                fetchedFromUrl = document.createElement('a');
+                fetchedFromUrl.setAttribute('href', input.value);
+                fetchedFromUrl.setAttribute('target', '_blank');
+                fetchedFromUrl.innerHTML = 'here';
```

Comment:
Otherwise looking great! I'll make this small suggestion:

```suggestion
                fetchedFromUrl.innerHTML = 'this Internet Archive collection';
```